manifests/init.pp

```diff
@@ -348,6 +353,7 @@
   Optional[Enum['yes', 'no']] $clear_node_high_bit                   = undef,
   Optional[Integer] $max_messages                                    = undef,
   Boolean $test_corosync_config                                      = $corosync::params::test_corosync_config,
+  Optional[String] $watchdog_device                                  = undef,
```

Comment:
Could you update this to something like:
```suggestion
  Optional[Variant[Stdlib::Absolutepath, Enum['off']]] $watchdog_device                                  = undef,
```

Reply:
Can do.